### PR TITLE
Beyza GOK week2 Databases

### PR DESCRIPTION
**Pull request by beyzagoknl**

### CODE REVIEW

File: Week2/HOMEWORK WEEK2/connection.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const keys_queries = require("./keys.js");
+const relationships_queries = require("./relationships.js");
+const joins_queries = require("./joins.js");
+const aggregate_queries = require("./aggregate.js");
+
+const connection = mysql.createConnection({
+  host: process.env.HOST,
+  user: process.env.USER,
+  password: process.env.PASSWORD,
+  database: process.env.DATABASE,
```
Comment (unmeshvrije) [inline Week2/HOMEWORK WEEK2/connection.js:13]: Great way of making the  connection !

File: Week2/HOMEWORK WEEK2/connection.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const mysql = require("mysql");
+const dotenv = require("dotenv");
+dotenv.config();
+const keys_queries = require("./keys.js");
+const relationships_queries = require("./relationships.js");
+const joins_queries = require("./joins.js");
+const aggregate_queries = require("./aggregate.js");
```
Comment (unmeshvrije) [inline Week2/HOMEWORK WEEK2/connection.js:7]: Good division of files according to the sub-sections of homework ! Well done 🎉

File: Week2/HOMEWORK WEEK2/relationships.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    ('What Is “Fake News” and How Does It Affect the Media?', 'M Conference', '2021-05-09'),
+    ('Is There Such a Thing as “Unbiased Reporting” and How Does One Find It?', 'O Sciences', '2022-04-12'),
+    ('What the Best Role Is for News Reporters in Our Digital Era and How to Adhere to that Role', 'P Conference', '2021-17-02'),
+    ('Can Brain Damage From Drugs Be Reversed and How?', 'L Conference', '2022-09-12'),
+    ('The Effects of Marijuana on the Brain', 'Art Conference', '2021-11-27'),
+    ('Can Learning to Do Lucid Dreaming Help You?', 'A Sciences', '2021-08-21'),
+    ('The day I left my childhood behind', 'Anthropological Conference', '2020-03-21'),
+    ('The power of music: The way heavy metal changed my life', 'Yes Conference', '2020-01-01'),
+    ('Why Buffy the vampire slayer is my hero', 'No Conference', '2022-03-22'),
+    ('The appropriate age to start discussing sex', 'Maybe Conference', '2021-01-01'),
```
Comment (unmeshvrije) [inline Week2/HOMEWORK WEEK2/relationships.js:58]: `Maybe Conference` is a nice name !

File: Week2/HOMEWORK WEEK2/keys.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+const queries = [
+    'DROP DATABASE IF EXISTS authors_data2',
+
+    'CREATE DATABASE IF NOT EXISTS authors_data2;',
+
+    'USE authors_data2;',
```
Comment (unmeshvrije) [inline Week2/HOMEWORK WEEK2/keys.js:6]: `authors_data2` is not a great name of the database, perhaps rename it to `research_db` or `scholars`

**Review by unmeshvrije — APPROVED** [review]
Great job! Please take into account the comments for the next homework

**Comment by beyzagoknl** [comment]
> Great job! Please take into account the comments for the next homework

@unmeshvrije Thank you for your time and your nice comments.

**Pull request closed without merging**